test/test_train_mp_imagenet.py

```diff
@@ -100,7 +102,7 @@ def train_imagenet():
         data=(torch.zeros(FLAGS.batch_size, 3, img_dim, img_dim),
               torch.zeros(FLAGS.batch_size, dtype=torch.int64)),
         sample_count=train_dataset_len // FLAGS.batch_size //
-            xm.xrt_world_size())
```

Comment:
Maybe try putting this line onto the line above:
`sample_count=train_dataset_len // FLAGS.batch_size // xm.xrt_world_size())`
and then running format again. My initial line breaking might be invalid and confused the formatter

Reply:
Same thing in test_train_imagenet.py

Reply:
We do not correct formatters. That''s the whole point of using them :wink:

Reply:
Sorry... misread you comment.
Formatter does the same thing, independently on where you put that line.
Otherwise it'd be non deterministic.